common/persistence/serialization/thrift_mapper.go

```diff
@@ -36,22 +36,22 @@ func shardInfoToThrift(info *ShardInfo) *sqlblobs.ShardInfo {
 		return nil
 	}
 	result := &sqlblobs.ShardInfo{
-		StolenSinceRenew:                      info.StolenSinceRenew,
-		ReplicationAckLevel:                   info.ReplicationAckLevel,
-		TransferAckLevel:                      info.TransferAckLevel,
-		DomainNotificationVersion:             info.DomainNotificationVersion,
+		StolenSinceRenew:                      &info.StolenSinceRenew,
+		ReplicationAckLevel:                   &info.ReplicationAckLevel,
+		TransferAckLevel:                      &info.TransferAckLevel,
+		DomainNotificationVersion:             &info.DomainNotificationVersion,
 		ClusterTransferAckLevel:               info.ClusterTransferAckLevel,
-		Owner:                                 info.Owner,
+		Owner:                                 &info.Owner,
 		ClusterReplicationLevel:               info.ClusterReplicationLevel,
 		PendingFailoverMarkers:                info.PendingFailoverMarkers,
-		PendingFailoverMarkersEncoding:        info.PendingFailoverMarkersEncoding,
+		PendingFailoverMarkersEncoding:        &info.PendingFailoverMarkersEncoding,
 		ReplicationDlqAckLevel:                info.ReplicationDlqAckLevel,
 		TransferProcessingQueueStates:         info.TransferProcessingQueueStates,
-		TransferProcessingQueueStatesEncoding: info.TransferProcessingQueueStatesEncoding,
+		TransferProcessingQueueStatesEncoding: &info.TransferProcessingQueueStatesEncoding,
 		TimerProcessingQueueStates:            info.TimerProcessingQueueStates,
-		TimerProcessingQueueStatesEncoding:    info.TimerProcessingQueueStatesEncoding,
-		UpdatedAtNanos:                        unixNanoPtr(info.UpdatedAt),
-		TimerAckLevelNanos:                    unixNanoPtr(info.TimerAckLevel),
+		TimerProcessingQueueStatesEncoding:    &info.TimerProcessingQueueStatesEncoding,
+		UpdatedAtNanos:                        common.Int64Ptr(info.UpdatedAt.UnixNano()),
```

Comment:
Any reason we don't use conversion helpers for time related fields like in many places below?